pkg/styra/users.go

```diff
@@ -29,16 +30,67 @@ const (
 	endpointV1Users = "/v1/users"
 )
 
-// GetUserResponse is the response type for calls to the GET /v1/users endpoint
+// GetUserResponse is the response type for calls to the GET /v1/users/{userId} endpoint
 // in the Styra API.
 type GetUserResponse struct {
 	StatusCode int
 	Body       []byte
 }
 
+// GetUsersResponse is the response type for calls to the GET /v1/users endpoint
+// in the Styra API.
+type GetUsersResponse struct {
+	Users []User
+}
+
+// Struct to unmarshal the JSON response from the GET /v1/users endpoint
+type getUsersJSONResponse struct {
```

Comment:
```suggestion
type GetUsersJSONResponse struct {
```
Is it on purpose that this is a small G?

Reply:
Yes :)